README.md

```diff
@@ -47,6 +47,28 @@ WebNN requires a compatible browser to run, and Windows 11 v21H2 (DML 1.6.0) or
 4. In the drop-down menu, select `Enabled`. 
 5. Relaunch your browser.
 
+#### Running WebNN on NPU
+Running WebNN on NPU requires capable NPU hardware e.g. Intel® AI Boost NPU of Intel® Core™ Ultra processors.
```

Comment:
Because developers always need to download and install NPU driver. To avoid duplication, you may want to pull the driver installation step here before the instructions for each browsers. Something like

### Install NPU driver
 
For Intel® AI Boost NPU of Intel® Core™ Ultra processors, download and install [the latest driver](https://www.intel.com/content/www/us/en/download/794734/intel-npu-driver-windows.html).

Other IHV NPU driver instruction can be added later.